diff --git a/oxidation/libparsec/crates/core_fs/src/schema.rs b/oxidation/libparsec/crates/core_fs/src/schema.rs
index 1181022e1..7f1e3f82f 100644
--- a/oxidation/libparsec/crates/core_fs/src/schema.rs
+++ b/oxidation/libparsec/crates/core_fs/src/schema.rs
@@ -1,6 +1,8 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
 table! {
     chunks (chunk_id) {
         chunk_id -> Binary,
         size -> Integer,
         offline -> Bool,
         accessed_on -> Nullable<Float>,
@@ -30,12 +32,7 @@ table! {
         remote_version -> Integer,
         need_sync -> Bool,
         blob -> Binary,
     }
 }

-allow_tables_to_appear_in_same_query!(
-    chunks,
-    prevent_sync_pattern,
-    realm_checkpoint,
-    vlobs,
-);
+allow_tables_to_appear_in_same_query!(chunks, prevent_sync_pattern, realm_checkpoint, vlobs,);
